### PR TITLE
change aboutus/banner agency components type

### PR DESCRIPTION
**Pull request by AMhaish**

### CODE REVIEW

File: lib/molecules/components/feature-block/index.tsx
Comment (AMhaish) [inline lib/molecules/components/feature-block/index.tsx:1]: Please let's keep on using single quotes while we are following that in other projects, I didn't apply the eslint rules here, will apply them soon
However no need to worry about this fine, just for the next pull requests

File: lib/containers/Agency/AboutUsSection/index.js
Comment (AMhaish) [inline lib/containers/Agency/AboutUsSection/index.js:1]: Please remove the files that you moved to organisms folder

File: lib/containers/organisms/components/BannerSections/AgencyBanner/bannerSection.style.ts
Comment (AMhaish) [inline lib/containers/organisms/components/BannerSections/AgencyBanner/bannerSection.style.ts:1]: Let's pick names that are not related to theme names, like I suggest here the next

lib/containers/organisms/components/banners/banner1/style.ts
lib/containers/organisms/components/banners/banner1/index.ts

And please don't forget to update
packages\showcase\lib\mappers\banners.js
To use the component from organisms instead of containers

File: lib/containers/organisms/components/BannerSections/AgencyBanner/index.tsx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  title,
+  btnStyle,
+  description,
+  discountText,
+  discountAmount,
+  outlineBtnStyle,
+  model,
+  isAR,
+}) => {
+  let data = model.data.reduce(function (map, obj) {
```
Comment (AMhaish) [inline lib/containers/organisms/components/BannerSections/AgencyBanner/index.tsx:24]:
```suggestion
  const data = model.data.reduce(function (map, obj) {
```
Let's convert it into const whenever we update a section, or whenever its suitable

File: lib/containers/organisms/components/BannerSections/AgencyBanner/index.tsx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+            ) : (
+              <i className="flaticon-next" />
+            )
+          }
+          {...outlineBtnStyle}
+        />
+      ) : null}
+    </Fragment>
+  );
+  return (
```
Comment (AMhaish) [inline lib/containers/organisms/components/BannerSections/AgencyBanner/index.tsx:58]: Let's keep empty line before return, also later I will add that rule to eslint file

File: lib/containers/organisms/components/BannerSections/AgencyBanner/index.tsx [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+BannerSection.propTypes = {
+  title: PropTypes.object,
+  btnStyle: PropTypes.object,
+  description: PropTypes.object,
+  contentStyle: PropTypes.object,
+  discountText: PropTypes.object,
+  discountAmount: PropTypes.object,
+  outlineBtnStyle: PropTypes.object,
+};
```
Comment (AMhaish) [inline lib/containers/organisms/components/BannerSections/AgencyBanner/index.tsx:104]: Let's convert propTypes to TS types, there is a cool tool for that, please check the next link:
https://mskelton.dev/ratchet

File: lib/containers/organisms/components/aboutUsSections/AgencyAboutus/index.tsx [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+};
+
+AboutUsSection.propTypes = {
+  row: PropTypes.object,
+  col: PropTypes.object,
+  textArea: PropTypes.object,
+  title: PropTypes.object,
+  description: PropTypes.object,
+  btnStyle: PropTypes.object,
+};
```
Comment (AMhaish) [inline lib/containers/organisms/components/aboutUsSections/AgencyAboutus/index.tsx:114]: Also, let's change here the propTypes to TS type and use it in components props

File: tsconfig.json [hunk 5]
```diff
@@ -13,6 +13,13 @@
...
   },
-  "include": ["**/*.ts", "**/*.tsx", "**/*.js", "**/*.jsx", "next-env.d.ts"],
+  "include": [
+    "**/*.ts",
+    "**/*.tsx",
+    "**/*.js",
+    "**/*.jsx",
+    "next-env.d.ts",
+    "lib/containers/organisms/components/FeatureSection/FeatureSection1/style.ts"
+, "lib/containers/organisms/components/QualitySections/QualitySection1/index.tsx"  ],
```
Comment (AMhaish) [inline tsconfig.json:23]: I think you have added these two lines by mistake

File: lib/containers/organisms/components/particals/Particle1/index.tsx
Comment (AMhaish) [inline lib/containers/organisms/components/particals/Particle1/index.tsx:1]: While this is not a section, and is just used inside the Agency banner, let's move it to the same folder of the banner
banners/banner1
We can rename it as particles.tsx there

File: lib/containers/organisms/components/navBarSections/Navbar1/index.tsx [hunk 6]
```diff
@@ -86,13 +94,13 @@ const Navbar = ({ navbarStyle, logoStyle, model, isAR }) => {
     });
   };
 
-  let data = model.data.reduce(function(map, obj) {
+  let data = model.data.reduce(function (map, obj) {
```
Comment (AMhaish) [inline lib/containers/organisms/components/navBarSections/Navbar1/index.tsx:97]:
```suggestion
  const data = model.data.reduce(function (map, obj) {
```

File: lib/containers/organisms/components/navBarSections/Navbar1/index.tsx [hunk 7]
```diff
@@ -86,13 +94,13 @@ const Navbar = ({ navbarStyle, logoStyle, model, isAR }) => {
...
     map[obj.Key] = obj.Value;
     return map;
   }, {});
   let socialModel;
   if (model.children && model.children.length > 0) {
-    let socialModelQuery = model.children.filter(m => m.modelId == 403193);
+    let socialModelQuery = model.children.filter((m) => m.modelId == 403193);
```
Comment (AMhaish) [inline lib/containers/organisms/components/navBarSections/Navbar1/index.tsx:103]:
```suggestion
    const socialModelQuery = model.children.filter((m) => m.modelId == 403193);
```

File: lib/containers/organisms/components/CopyrightsSections/CopyrightsSection1/index.tsx [hunk 8]
```diff
@@ -1,15 +1,14 @@
...
   return (
     <CopyrightWrapper className="copyright_section">
       <ul>
         {model.children &&
           model.children
-            .filter(m => m.online)
+            .filter((m) => m.online)
             .map((profileItem, index) => {
-              let profile = profileItem.data.reduce(function(map, obj) {
+              let profile = profileItem.data.reduce(function (map, obj) {
```
Comment (AMhaish) [inline lib/containers/organisms/components/CopyrightsSections/CopyrightsSection1/index.tsx:11]:
```suggestion
              const profile = profileItem.data.reduce(function (map, obj) {
```

File: lib/containers/organisms/components/CopyrightsSections/CopyrightsSection1/index.tsx
Comment (AMhaish) [inline lib/containers/organisms/components/CopyrightsSections/CopyrightsSection1/index.tsx:1]: While this is not a section, let's move it inside navbars/navbar1, cause it will only be used there

File: lib/containers/organisms/components/CopyrightsSections/CopyrightsSection1/style.ts
Comment (AMhaish) [inline lib/containers/organisms/components/CopyrightsSections/CopyrightsSection1/style.ts:1]: Same here let's move it under navbars/navbar1

File: lib/containers/organisms/components/FaqSection/FaqSection1/index.tsx
Comment (AMhaish) [inline lib/containers/organisms/components/FaqSection/FaqSection1/index.tsx:1]: let's rename the path as next
faqs/faq1

File: lib/containers/organisms/components/Footers/Footer1/index.tsx
Comment (AMhaish) [inline lib/containers/organisms/components/Footers/Footer1/index.tsx:1]: let's rename it as
footers/footer1

File: lib/containers/organisms/components/LogInModals/LoginModal/index.tsx
Comment (AMhaish) [inline lib/containers/organisms/components/LogInModals/LoginModal/index.tsx:1]: While this is not a section, also we might use it in other templates, let's move it to the molecules folder

File: lib/containers/organisms/components/NewsletterSections/NewsletterSection1/index.tsx
Comment (AMhaish) [inline lib/containers/organisms/components/NewsletterSections/NewsletterSection1/index.tsx:1]: Let's rename it as
newsletters/newsletter1

File: lib/containers/organisms/components/QualitySections/QualitySection1/index.tsx
Comment (AMhaish) [inline lib/containers/organisms/components/QualitySections/QualitySection1/index.tsx:1]: I know they are calling it as Quality Section, but we need to be generic in naming our sections
I think this one and the second section "OUR SERVICES" can be listed under featuresList folder like:
featuresList/featureList1
featuresList/featureList2

File: lib/containers/organisms/components/SearchPanels/SearchPanel1/index.tsx
Comment (AMhaish) [inline lib/containers/organisms/components/SearchPanels/SearchPanel1/index.tsx:1]: While this is not a section, also we might use it in other templates, let's move it to the molecules folder

File: lib/containers/organisms/components/TakeActionSections/TakeActionSection1/index.tsx
Comment (AMhaish) [inline lib/containers/organisms/components/TakeActionSections/TakeActionSection1/index.tsx:1]: Let's rename it as:
callToActions/callToAction1
Usually "Call to action" is more common word between UX people

File: lib/containers/organisms/components/TeamSections/TeamSection1/index.tsx
Comment (AMhaish) [inline lib/containers/organisms/components/TeamSections/TeamSection1/index.tsx:1]: let's rename it as:
teamLists/teamList1

File: lib/containers/organisms/components/TestimonialSections/TestimonialSection1/index.tsx
Comment (AMhaish) [inline lib/containers/organisms/components/TestimonialSections/TestimonialSection1/index.tsx:1]: let's rename it as
testimonials/testimonial1

File: lib/containers/organisms/components/blogSections/blogSection1/index.tsx
Comment (AMhaish) [inline lib/containers/organisms/components/blogSections/blogSection1/index.tsx:1]: let's rename it as:
portfolios/portfolio1

File: lib/containers/organisms/components/blogSliderSection/blogSlider1/index.tsx
Comment (AMhaish) [inline lib/containers/organisms/components/blogSliderSection/blogSlider1/index.tsx:1]: let's rename the path as:
sliders/slider1

File: lib/molecules/components/glide-carousel/glideSlide.tsx [hunk 9]
```diff
@@ -2,8 +2,12 @@ import React from 'react';
...
-    <GlideSlideWrapper className="glide__slide">{children}</GlideSlideWrapper>
-  );
+export const GlideSlide = ({
+  children,
+  props,
+}: {
+  children: React.ReactNode;
+  props?: React.HTMLProps<HTMLLIElement>;
+}) => {
+  return <GlideSlideWrapper {...{ props }}>{children}</GlideSlideWrapper>;
```
Comment (AMhaish) [inline lib/molecules/components/glide-carousel/glideSlide.tsx:12]: Why we needed these changes, and removing glide__slide class didn't effect the style?

File: lib/containers/organisms/components/aboutUsSections/aboutUsSection1/index.tsx
Comment (AMhaish) [inline lib/containers/organisms/components/aboutUsSections/aboutUsSection1/index.tsx:1]: I think also we can move this section to
featuresLists/featureList3

It's kinda feature list for me too

**Review by AMhaish — COMMENTED** [review]
Please also let's make sure the template is working fine locally

File: lib/containers/organisms/components/CopyrightsSections/CopyrightsSection1/index.tsx
Comment (AMhaish) [reply lib/containers/organisms/components/CopyrightsSections/CopyrightsSection1/index.tsx:1]: @AutoZoneMostafa reminder I think you missed this comment

File: lib/containers/organisms/components/CopyrightsSections/CopyrightsSection1/index.tsx (hunk 8 above)
Comment (AMhaish) [reply lib/containers/organisms/components/CopyrightsSections/CopyrightsSection1/index.tsx:11]: @AutoZoneMostafa reminder I think you missed this comment

File: lib/containers/organisms/components/CopyrightsSections/CopyrightsSection1/style.ts
Comment (AMhaish) [reply lib/containers/organisms/components/CopyrightsSections/CopyrightsSection1/style.ts:1]: @AutoZoneMostafa I think you missed this comment

File: lib/containers/organisms/components/FaqSection/FaqSection1/index.tsx
Comment (AMhaish) [reply lib/containers/organisms/components/FaqSection/FaqSection1/index.tsx:1]: @AutoZoneMostafa I think you missed this one

File: lib/containers/organisms/components/Footers/Footer1/index.tsx
Comment (AMhaish) [reply lib/containers/organisms/components/Footers/Footer1/index.tsx:1]: @AutoZoneMostafa I think you missed this one

File: lib/containers/organisms/components/LogInModals/LoginModal/index.tsx
Comment (AMhaish) [reply lib/containers/organisms/components/LogInModals/LoginModal/index.tsx:1]: @AutoZoneMostafa I think you missed this one

File: lib/containers/organisms/components/NewsletterSections/NewsletterSection1/index.tsx
Comment (AMhaish) [reply lib/containers/organisms/components/NewsletterSections/NewsletterSection1/index.tsx:1]: @AutoZoneMostafa I think you missed this one

File: lib/containers/organisms/components/QualitySections/QualitySection1/index.tsx
Comment (AMhaish) [reply lib/containers/organisms/components/QualitySections/QualitySection1/index.tsx:1]: @AutoZoneMostafa I think you missed this one

File: lib/containers/organisms/components/SearchPanels/SearchPanel1/index.tsx
Comment (AMhaish) [reply lib/containers/organisms/components/SearchPanels/SearchPanel1/index.tsx:1]: @AutoZoneMostafa I think you missed this one

File: lib/containers/organisms/components/TakeActionSections/TakeActionSection1/index.tsx
Comment (AMhaish) [reply lib/containers/organisms/components/TakeActionSections/TakeActionSection1/index.tsx:1]: @AutoZoneMostafa I think you missed this one

File: lib/containers/organisms/components/TeamSections/TeamSection1/index.tsx
Comment (AMhaish) [reply lib/containers/organisms/components/TeamSections/TeamSection1/index.tsx:1]: @AutoZoneMostafa I think you missed this one

File: lib/containers/organisms/components/TestimonialSections/TestimonialSection1/index.tsx
Comment (AMhaish) [reply lib/containers/organisms/components/TestimonialSections/TestimonialSection1/index.tsx:1]: @AutoZoneMostafa I think you missed this one

File: lib/containers/organisms/components/aboutUsSections/aboutUsSection1/index.tsx
Comment (AMhaish) [reply lib/containers/organisms/components/aboutUsSections/aboutUsSection1/index.tsx:1]: @AutoZoneMostafa I think you missed this one

File: lib/containers/organisms/components/blogSections/blogSection1/index.tsx
Comment (AMhaish) [reply lib/containers/organisms/components/blogSections/blogSection1/index.tsx:1]: @AutoZoneMostafa I think you missed this one

File: lib/containers/organisms/components/blogSliderSection/blogSlider1/index.tsx
Comment (AMhaish) [reply lib/containers/organisms/components/blogSliderSection/blogSlider1/index.tsx:1]: @AutoZoneMostafa I think you missed this one

File: lib/containers/organisms/components/navBarSections/Navbar1/index.tsx (hunk 6 above)
Comment (AMhaish) [reply lib/containers/organisms/components/navBarSections/Navbar1/index.tsx:97]: @AutoZoneMostafa I think you missed this one

File: lib/containers/organisms/components/navBarSections/Navbar1/index.tsx (hunk 7 above)
Comment (AMhaish) [reply lib/containers/organisms/components/navBarSections/Navbar1/index.tsx:103]: @AutoZoneMostafa I think you missed this one

File: lib/containers/organisms/components/navBarSections/Navbar1/navbar.style.ts
[the remark this answers is not in the document]
Comment (AMhaish) [reply lib/containers/organisms/components/navBarSections/Navbar1/navbar.style.ts:1]: @AutoZoneMostafa I think you missed this one

File: lib/containers/organisms/components/particals/Particle1/index.tsx
Comment (AMhaish) [reply lib/containers/organisms/components/particals/Particle1/index.tsx:1]: @AutoZoneMostafa I think you missed this one

File: lib/mappers/banners.js
Comment (AMhaish) [inline lib/mappers/banners.js:1]: @AutoZoneMostafa while I changed mappers place, I think you need to pull develop to your branch and do the needed updates

File: lib/mappers/horizontalSliders.js
Comment (AMhaish) [inline lib/mappers/horizontalSliders.js:1]: @AutoZoneMostafa while I changed mappers place, I think you need to pull develop to your branch and do the needed updates

File: lib/molecules/components/glide-carousel/glideSlide.tsx (hunk 9 above)
Comment (AMhaish) [reply lib/molecules/components/glide-carousel/glideSlide.tsx:12]: @AutoZoneMostafa I think you missed this one

File: tsconfig.json (hunk 5 above)
Comment (AMhaish) [reply tsconfig.json:23]: @AutoZoneMostafa I think you missed this one

**Review by AMhaish — COMMENTED** [review]
@AutoZoneMostafa also please test the template as whole to make sure it's working fine as we have it in the next link

http://showcase-agency.bcrumbs.net/

File: lib/containers/organisms/components/blogSliderSection/blogSlider1/index.tsx
Comment (RanimHelamiah) [reply lib/containers/organisms/components/blogSliderSection/blogSlider1/index.tsx:1]: done

File: lib/containers/organisms/components/blogSections/blogSection1/index.tsx
Comment (RanimHelamiah) [reply lib/containers/organisms/components/blogSections/blogSection1/index.tsx:1]: done

File: lib/containers/organisms/components/TestimonialSections/TestimonialSection1/index.tsx
Comment (RanimHelamiah) [reply lib/containers/organisms/components/TestimonialSections/TestimonialSection1/index.tsx:1]: done

File: lib/containers/organisms/components/TeamSections/TeamSection1/index.tsx
Comment (RanimHelamiah) [reply lib/containers/organisms/components/TeamSections/TeamSection1/index.tsx:1]: done

File: lib/containers/organisms/components/TakeActionSections/TakeActionSection1/index.tsx
Comment (RanimHelamiah) [reply lib/containers/organisms/components/TakeActionSections/TakeActionSection1/index.tsx:1]: done

File: lib/containers/organisms/components/NewsletterSections/NewsletterSection1/index.tsx
Comment (RanimHelamiah) [reply lib/containers/organisms/components/NewsletterSections/NewsletterSection1/index.tsx:1]: done

File: lib/containers/organisms/components/Footers/Footer1/index.tsx
Comment (RanimHelamiah) [reply lib/containers/organisms/components/Footers/Footer1/index.tsx:1]: done

File: lib/containers/organisms/components/FaqSection/FaqSection1/index.tsx
Comment (RanimHelamiah) [reply lib/containers/organisms/components/FaqSection/FaqSection1/index.tsx:1]: done

File: lib/containers/organisms/components/CopyrightsSections/CopyrightsSection1/style.ts
Comment (RanimHelamiah) [reply lib/containers/organisms/components/CopyrightsSections/CopyrightsSection1/style.ts:1]: done

File: lib/containers/organisms/components/CopyrightsSections/CopyrightsSection1/index.tsx
Comment (RanimHelamiah) [reply lib/containers/organisms/components/CopyrightsSections/CopyrightsSection1/index.tsx:1]: done

File: lib/containers/organisms/components/blogSliderSection/blogSlider1/index.tsx [hunk 10]
```diff
@@ -43,13 +51,13 @@ const BlogSliderSection = ({
       },
     },
   };
-  let data = model.data.reduce(function(map, obj) {
+  let data = model.data.reduce(function (map, obj) {
```
[the remark this answers is not in the document]
Comment (RanimHelamiah) [reply lib/containers/organisms/components/blogSliderSection/blogSlider1/index.tsx:54]: done

File: lib/containers/organisms/components/navBarSections/Navbar1/index.tsx (hunk 7 above)
Comment (RanimHelamiah) [reply lib/containers/organisms/components/navBarSections/Navbar1/index.tsx:103]: done

File: lib/containers/organisms/components/navBarSections/Navbar1/index.tsx (hunk 6 above)
Comment (RanimHelamiah) [reply lib/containers/organisms/components/navBarSections/Navbar1/index.tsx:97]: done

File: lib/containers/organisms/components/navBarSections/Navbar1/navbar.style.ts
[the remark this answers is not in the document]
Comment (RanimHelamiah) [reply lib/containers/organisms/components/navBarSections/Navbar1/navbar.style.ts:1]: done

File: lib/containers/organisms/components/particals/Particle1/index.tsx
Comment (RanimHelamiah) [reply lib/containers/organisms/components/particals/Particle1/index.tsx:1]: done

File: tsconfig.json (hunk 5 above)
Comment (RanimHelamiah) [reply tsconfig.json:23]: done

File: lib/containers/organisms/components/LogInModals/LoginModal/index.tsx
Comment (RanimHelamiah) [reply lib/containers/organisms/components/LogInModals/LoginModal/index.tsx:1]: done

File: lib/containers/organisms/components/QualitySections/QualitySection1/index.tsx
Comment (RanimHelamiah) [reply lib/containers/organisms/components/QualitySections/QualitySection1/index.tsx:1]: done

File: lib/containers/organisms/components/SearchPanels/SearchPanel1/index.tsx
Comment (RanimHelamiah) [reply lib/containers/organisms/components/SearchPanels/SearchPanel1/index.tsx:1]: done

File: lib/containers/organisms/components/aboutUsSections/aboutUsSection1/index.tsx
Comment (RanimHelamiah) [reply lib/containers/organisms/components/aboutUsSections/aboutUsSection1/index.tsx:1]: done

File: lib/molecules/components/glide-carousel/glideSlide.tsx (hunk 9 above)
Comment (RanimHelamiah) [reply lib/molecules/components/glide-carousel/glideSlide.tsx:12]: I will leave develop changes

File: lib/organisms/components/portfolios/portfolio1/index.tsx [hunk 11]
```diff
@@ -1,18 +1,20 @@
...
-  Container
-} from '../../../atoms';
-import { FeatureBlock } from '../../../molecules';
-
-import BlogSectionWrapper from './blogSection.style';
-
+import BlogSectionWrapper from './style';
+import { Container, Box, Heading, Button,Text,Image,Link } from '../../../../atoms';
+import { FeatureBlock } from '../../../../molecules';
+interface IBlogSliderSection {
```
Comment (AMhaish) [inline lib/organisms/components/portfolios/portfolio1/index.tsx:6]:
```suggestion
interface BlogSliderSectionProps {
```

File: lib/organisms/components/sliders/slider1/index.tsx [hunk 12]
```diff
@@ -1,19 +1,28 @@
...
   TextWrapper,
   CarouselWrapper,
-} from './blogSliderSection.style';
-
+} from './style';
+import { Container, Box, Heading, Button , Image} from '../../../../atoms';
+import { GlideCarousel, GlideSlide } from '../../../../molecules';
+// import { Box, Heading, Image, Button, Container } from '../../../../../atoms';
+// import { GlideCarousel, GlideSlide } from '../../../../../molecules';
+interface IBlogSliderSection {
```
Comment (AMhaish) [inline lib/organisms/components/sliders/slider1/index.tsx:16]:
```suggestion
interface BlogSliderSectionProps {
```

File: lib/organisms/components/testimonials/testimonial1/index.tsx [hunk 13]
```diff
@@ -0,0 +1,191 @@
...
+  sectionTitle: PropTypes.object,
+  sectionSubTitle: PropTypes.object,
+  row: PropTypes.object,
+  col: PropTypes.object,
+  btnStyle: PropTypes.object,
+  btnWrapperStyle: PropTypes.object,
+  nameStyle: PropTypes.object,
+  commentStyle: PropTypes.object,
+  designationStyle: PropTypes.object,
+};
```
Comment (AMhaish) [inline lib/organisms/components/testimonials/testimonial1/index.tsx:120]: Let's please convert this into s TS interface

File: lib/organisms/components/testimonials/testimonial1/index.tsx [hunk 14]
```diff
@@ -0,0 +1,191 @@
...
+  model,
+}) => {
+  // Glide js options
+  const glideOptions = {
+    type: 'carousel',
+    autoplay: 5000,
+    perView: 1,
+    animationDuration: 700,
+  };
+  let data = model.data.reduce(function (map, obj) {
```
Comment (AMhaish) [inline lib/organisms/components/testimonials/testimonial1/index.tsx:29]:
```suggestion
  const data = model.data.reduce(function (map, obj) {
```

File: bootstrapers/showcase/mappers/banners.js [hunk 15]
```diff
@@ -2,7 +2,7 @@ import PortfolioBannerSection from '../../../lib/containers/Portfolio/Banner';
 import SaaSBannerSection from '../../../lib/containers/Saas/BannerSection';
 import RideBannerSecion from '../../../lib/containers/Ride/Banner';
 import CharityBannerSection from '../../../lib/containers/Charity/BannerSection';
-import AgencyBannerSection from '../../../lib/containers/Agency/BannerSection';
+import BannerSection from '../../../lib/organisms/components/banners/banner1';
```
[the remark this answers is not in the document]
Comment (RanimHelamiah) [reply bootstrapers/showcase/mappers/banners.js:5]: done

File: bootstrapers/showcase/mappers/contact.js [hunk 16]
```diff
@@ -1,6 +1,6 @@
 import PortfolioContactSection from '../../../lib/containers/Portfolio/Contact';
 import FoodContactSection from '../../../lib/containers/Food/ContactUsSection';
-import TakeActionSection from '../../../lib/containers/Agency/TakeActionSection';
+import TakeActionSection from '../../../lib/organisms/components/callToActions/callToAction1';
```
[the remark this answers is not in the document]
Comment (RanimHelamiah) [reply bootstrapers/showcase/mappers/contact.js:3]: done

File: lib/molecules/LogInModals/LogInModal/index.tsx
[the remark this answers is not in the document]
Comment (RanimHelamiah) [reply lib/molecules/LogInModals/LogInModal/index.tsx:1]: done

File: lib/molecules/LogInModals/LogInModal/index.tsx [hunk 17]
```diff
@@ -65,7 +67,7 @@ const LoginModal = ({
                   {...descriptionStyle}
                 />
                 <Button
-                  icon={<Image src={GoogleLogo} alt="Google Icon" />}
+                  icon={<Image src={GoogleLogo.src} alt="Google Icon" />}
```
[the remark this answers is not in the document]
Comment (RanimHelamiah) [reply lib/molecules/LogInModals/LogInModal/index.tsx:70]: we don't need

File: lib/molecules/SearchPanels/SearchPanel1/index.tsx
[the remark this answers is not in the document]
Comment (RanimHelamiah) [reply lib/molecules/SearchPanels/SearchPanel1/index.tsx:1]: done

File: lib/organisms/components/navbars/navbar1/index.tsx [hunk 18]
```diff
@@ -29,8 +26,14 @@ import CopyrightSection from '../CopyrightsSection';
 //     icon={<i className="flaticon-plus-symbol" />}
 //   />
 // );
+interface INavbar {
```
[the remark this answers is not in the document]
Comment (RanimHelamiah) [reply lib/organisms/components/navbars/navbar1/index.tsx:29]: done

File: lib/organisms/components/portfolios/portfolio1/index.tsx (hunk 11 above)
Comment (RanimHelamiah) [reply lib/organisms/components/portfolios/portfolio1/index.tsx:6]: done

File: lib/organisms/components/sliders/slider1/index.tsx (hunk 12 above)
Comment (RanimHelamiah) [reply lib/organisms/components/sliders/slider1/index.tsx:16]: done

File: lib/organisms/components/testimonials/testimonial1/index.tsx (hunk 14 above)
Comment (RanimHelamiah) [reply lib/organisms/components/testimonials/testimonial1/index.tsx:29]: done

File: lib/molecules/LogInModals/LogInModal/index.tsx (hunk 17 above)
[the remark this answers is not in the document]
Comment (RanimHelamiah) [reply lib/molecules/LogInModals/LogInModal/index.tsx:70]: we need it, without it, it gives an error..
But I didn't add the suffix

File: lib/organisms/components/testimonials/testimonial1/index.tsx (hunk 13 above)
Comment (AMhaish) [reply lib/organisms/components/testimonials/testimonial1/index.tsx:120]: @RanimHelamiah I think you missed this comment Ranim :)

Comment (RanimHelamiah) [reply lib/organisms/components/testimonials/testimonial1/index.tsx:135]: Sorry, I did it
